src/Model/Table/QueuedJobsTable.php

```diff
@@ -983,4 +983,21 @@ protected function getDateTime($notBefore = null) {
 		return new FrozenTime($notBefore);
 	}
 
+	/**
+	 * Sends a SIGUSR1 to all workers of this server.
+	 *
+	 * @return void
+	 */
+	public function wakeUpWorkers() {
+		if (!function_exists('posix_kill')) {
+			return;
+		}
+		$processes = $this->getProcesses(true);
```

Comment:
@dereuromark Actually I am facing a problem with this parameter `true` and I am not sure what’s the best way to solve it. I thought it would make sense to use `true` to restrict the processes to the ones running on the same server. However, it doesn’t seem to work as expected, because `buildServerString()` returns the fully qualified host name (such as www.example.com) when running in php-fpm, whereas it returns the "internal" host name (often showing on the shell prompt) when running from the cake console. As a result, the CGI code mistakenly assumes the worker process is running on different machine and avoids sending the signal.

Should I just remove the `true` parameter?

Reply:
Hm, I guess so

Reply:
Thanks for confirming. I opened a new PR #273. Hopefully this will be the last one for that feature :sweat_smile: